packages/app/src/components/HyperJson.tsx

```diff
@@ -177,6 +195,25 @@ const Line = React.memo(
       [keyName, parentKeyPath],
     );
 
+    // Hide LineMenu when selecting text in the value
+    const valueRef = React.useRef<HTMLSpanElement>(null);
+    const [isSelectingValue, setIsSelectingValue] = React.useState(false);
+    const handleValueSelectStart = React.useCallback(() => {
+      setIsSelectingValue(true);
+    }, []);
+    const handleValueMouseUp = React.useCallback(() => {
+      setIsSelectingValue(false);
+    }, []);
+    React.useEffect(() => {
+      const _valueRef = valueRef.current;
```

Comment:
Should this be optionally chained since valueRef can be null? or is that case impossible

Reply:
i believe it can't be null since it comes from useRef, but only .current can be null?

Reply:
No I think you're right, I still mess up useref hooks lifecycle 😓